pyradiosky/skymodel.py

```diff
@@ -4305,6 +3849,22 @@ def read_gleam_catalog(
             freq_array = [76, 84, 92, 99, 107, 115, 122, 130, 143, 151, 158, 166,
                           174, 181, 189, 197, 204, 212, 220, 227]
             freq_array = np.array(freq_array) * 1e6 * units.Hz
+
+            # GLEAM paper specifies that the 30.72 MHz bandwidth is subdivided into
+            # four 7.68 MHz sub-channels
+            # But that clashes with the frequencies and edges listed in the catalog
+            # documentation. Going with the catalog docs for now, but deeply uncertain.
```

Comment:
I think this is the right choice.  Just debating whether we need to present this to the user in a slightly more obvious way, as this comment is buried pretty deep in the read_gleam_catalog method.  How awful would it be to have a keyword that defaults to using these band edges, but can be set to False to use the paper values instead?

Reply:
yes, will do.

Reply:
I added an option: `use_paper_freqs` which calculates the values from what I know about MWA. The band centers using this option are different from the catalog values by at most 0.6 MHz. The band edges are different by at most 1.08 MHz.